README.rst

```diff
@@ -223,15 +235,15 @@ In general, this should work (assuming CUDA 9):
 .. code:: bash
 
     # using conda:
-    conda install pytorch -c pytorch
+    conda install pytorch cudatoolkit=9.0 -c pytorch
```

Comment:
Do we want users on 10.0?

Reply:
PyTorch uses 9 as the default, so we'll stick with 9 for now, I guess.